arangod/RocksDBEngine/RocksDBReplicationManager.cpp

```diff
@@ -305,6 +306,28 @@ void RocksDBReplicationManager::drop(TRI_vocbase_t* vocbase) {
   garbageCollect(false);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief drop contexts by collection (at least mark them as deleted)
+////////////////////////////////////////////////////////////////////////////////
+
+void RocksDBReplicationManager::drop(LogicalCollection* collection) {
+  LOG_TOPIC("fe4bb", TRACE, Logger::REPLICATION)
```

Comment:
I know it is not called like this, but I would like to see an ASSERTION that collection is not a nullptr.
Just to be save in future

Reply:
As this PR is for 3.6, I will add the assertion to devel now.

Reply:
Added via 425d3fd0d9

Reply:
Nice me much like :+1: